reduction/imaging_parameters.py

```diff
@@ -3121,11 +3127,11 @@
     },
     "W43-MM2": {
         "12co": {"cubewidth": "150km/s"},
-        "sio": {"cubewidth": "60km/s", "vlsr": "90km/s"},
```

Comment:
this shouldn't be removed, right?

Reply:
I think it should because it's defined again a few lines later.

Reply:
ah, yes, I see!